VALIDATION.md

```diff
@@ -0,0 +1,46 @@
+# Dance Party: Validation Feedback Documentation
+
+Dance levels can include validation feedback configured with `validation_code`. There is a general consensus that many students do not read level instructions carefully so that validation feedback is critical for students to engage successfully with a given activity.
+
+Currently (January 2024), there are two versions of Dance Party progressions:
+- [Dance Party (2019)](https://studio.code.org/s/dance-2019) - [script](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts_json/dance-2019.script_json)
+- [Dance Party: AI Edition (2023)](https://studio.code.org/s/dance-ai-2023) - [script](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts_json/dance-ai-2023.script_json)
+
+For the 2023 version of Dance Party, additional support was added to `code-dot-org/dance-party` to provide more targeted feedback for users. Below are functions and variables that are used in level `validation_code` added for Dance Party: AI Edition.
+
+| Name | Description |
```

Comment:
Just an idea - would it be helpful to include the version of dance party supported as a separate column and combine these two lists?
```
| Name | Description | Supported Versions |
| getCurrentPalette() | A function ... | Dance AI |
...
| pass() | A function ... | Dance AI, Dance 2019 |
```

Reply:
oh yeah! I like the second option. Will update.